archlinuxcn/video-trimmer/PKGBUILD

```diff
@@ -0,0 +1,36 @@
+# Maintainer: torculus <hidden> 
+# Contributor: Mark Wagie <mark dot wagie at proton dot me>
+pkgname=video-trimmer
+pkgver=0.8.2
+pkgrel=1
+pkgdesc="Trim videos quickly"
+arch=('x86_64')
+url="https://apps.gnome.org/VideoTrimmer"
+license=('GPL3')
+depends=('ffmpeg' 'gst-plugins-good' 'libadwaita')
+makedepends=('blueprint-compiler' 'cargo' 'git' 'meson')
```

Comment:
```
video-trimmer W: Dependency hicolor-icon-theme detected and implicitly satisfied (needed for hicolor theme hierarchy)
video-trimmer W: Dependency dconf detected and implicitly satisfied (needed for glib schemas)
video-trimmer W: Dependency glib2 detected and implicitly satisfied (libraries ['usr/lib/libglib-2.0.so.0', 'usr/lib/libgio-2.0.so.0', 'usr/lib/libgobject-2.0.so.0'] needed in files ['usr/bin/video-trimmer'])
video-trimmer W: Dependency gtk4 detected and implicitly satisfied (libraries ['usr/lib/libgtk-4.so.1'] needed in files ['usr/bin/video-trimmer'])
video-trimmer W: Dependency glibc detected and implicitly satisfied (libraries ['usr/lib/ld-linux-x86-64.so.2', 'usr/lib/libc.so.6', 'usr/lib/libm.so.6'] needed in files ['usr/bin/video-trimmer'])
video-trimmer W: Dependency gdk-pixbuf2 detected and implicitly satisfied (libraries ['usr/lib/libgdk_pixbuf-2.0.so.0'] needed in files ['usr/bin/video-trimmer'])
video-trimmer W: Dependency cairo detected and implicitly satisfied (libraries ['usr/lib/libcairo.so.2'] needed in files ['usr/bin/video-trimmer'])
video-trimmer W: Dependency gcc-libs detected and implicitly satisfied (libraries ['usr/lib/libgcc_s.so.1'] needed in files ['usr/bin/video-trimmer'])
video-trimmer W: Dependency pango detected and implicitly satisfied (libraries ['usr/lib/libpango-1.0.so.0'] needed in files ['usr/bin/video-trimmer'])
video-trimmer W: Dependency included, but may not be needed ('ffmpeg')
video-trimmer W: Dependency included, but may not be needed ('gst-plugins-good')
```

Reply:
Added the first 9 implicitly satisfied dependencies.
But ffmpeg and gst-plugins-good should be included refer to upstream'[ readme](https://gitlab.gnome.org/YaLTeR/video-trimmer/-/blob/master/README.md?ref_type=heads#format-support)